README.md

```diff
@@ -54,19 +53,10 @@ Once you have the application running locally, in a separate terminal window, ma
 
 You should see a dynamically generated public Ngrok URL in the command window. Ngrok will now tunnel all HTTP traffic directed at this URL to your local machine at port 3000.
 
-### Deploy to the cloud using [Heroku](https://heroku.com)
+### Test the app
 
-Don't want to run the application server locally? You can also deploy it to the cloud using Heroku. Note: For this option, you'll want to gather your Twilio Account SID, API Key, API Key Secret, Push Credential SID and TwiML App SID before beginning the installation. (Don't know what these are? Read through the [Programmable Voice iOS SDK Quickstart](https://www.twilio.com/docs/api/voice-sdk/ios/getting-started) or [Programmable Voice Android SDK Quickstart](https://www.twilio.com/docs/api/voice-sdk/android/getting-started) for all the information.
+Test your app by opening the `{NGROK_SERVER_URL}/accessToken` endpoint in your browser. Use the publicly accessible domain on ngrok. You should see a long string. This is an Access Token. You can examine its contents by pasting it into a JWT tool like [jwt.io](http://jwt.io).
```

Comment:
Now that we already have `NGROK_SERVER_URL` I think it's okay to remove `Use the publicly accessible domain on ngrok`?